docs/remove_rhts.rst

```diff
@@ -1,3 +1,5 @@
+.. _rm_rhts_guide:
```

Comment:
This shouldn't be part of this PR.

Reply:
I'm fine with adding this one. There is a ref in notes

Reply:
Reference is for #27, at least this should be in a separate commit.

Reply:
I was missing that at the moment this is only used by release notes, so indeed, this makes sense in this PR.
Personally, I would put this on a different commit, but I leave that decision to OP.

Reply:
Since the reference to this ref point is within the same release notes, I believe it belongs in this changeset.